config/default/common/config/wv.json/stories/default/worldview_layerpicker.json

```diff
@@ -0,0 +1,189 @@
+{
+  "stories": {
+    "worldview_layerpicker": {
+      "id": "worldview_layerpicker",
+      "type": "default",
+      "title": "Worldview's Layer Picker",
+      "description": "Discover all the imagery and imagery visualizations available in Worldview!",
+      "backgroundImage": "",
+      "backgroundImageHover": "",
+      "readMoreLinks": [],
+      "steps": [
+        {
+          "id": "001",
+          "description": "step001.html",
+          "stepLink": "v=-108.00920361862684,-63.57842410029712,116.2759949373298,68.07713930391033&lg=false&tr=worldview_intro&t=2021-03-22-T16%3A21%3A03Z",
+          "joyride": {
+            "continuous": true,
+            "spotlightClicks": true,
+            "steps": [
+              {
+                "target": "#productsHolder",
+                "content": "To start, let's explore the Layers, Events and Data tabs.",
+                "placement": "right-start"
+              },
+              {
+                "target": "#layers-sidebar-tab",
+                "title": "Layer List",
+                "content": "The Layer List shows all the imagery layers that are loaded on the map. Base layers under the overlays and only the top most activated base layer will display.",
+                "placementBeacon": "top"
+              },
+              {
+                "target": "#events-sidebar-tab",
+                "title": "Events",
+                "content": "Select the Events tab to view a curated list of current natural events around the world.",
+                "placementBeacon": "right"
+              },
+              {
+                "target": "#download-sidebar-tab",
+                "title": "Data Download",
+                "content": "Select the Data tab to download the underlying data for the imagery layers displayed on the map. Once you have selected your desired layer it will direct you to Earthdata Search to download the data.",
```

Comment:
I changed it to "Select the Data tab to download the underlying data for the imagery layers displayed on the map. You will be directed to the Earthdata Search tool to download data granules of the selected layer, chosen area of interest, and current date.", since users may not know what Earthdata Search is.